### DISCUSSION TITLE
Validate user emails

### DISCUSSION THREAD

**Discussion by Priscilaboldo** [community/community#50042] [Other Feature Feedback, Questions, & Ideas]
Good afternoon,

I would like to know how do I validate the emails of each user? Where can I "see" this information?
How can I tell if an organization user is using a personal or corporate account?

**Comment by neutrinomuon** [comment]
Hello,

As far as I know, there is no specific feature on GitHub that allows you to directly determine whether a user is using a personal or corporate account. However, you can often make an educated guess based on the user's profile information and email address.

As mentioned before, if a user has listed their employer or organization on their profile and their email address is a corporate email address, it is likely that they are using a corporate account. On the other hand, if a user has not listed an employer or organization on their profile and their email address is a personal email address, it is more likely that they are using a personal account.

But it's worth noting that this is not a foolproof method, and there are cases where users may use personal email addresses for their corporate accounts or vice versa. So, it's best to use this information as a general guide and not rely on it entirely.

I hope this helps.
